ngraph/python/BUILDING.md

```diff
@@ -51,64 +48,63 @@ set the mentioned flags to `ON`. Note the `CMAKE_INSTALL_PREFIX`, which defaults
         -DENABLE_PYTHON=ON \
         -DNGRAPH_PYTHON_BUILD_ENABLE=ON \
         -DNGRAPH_ONNX_IMPORT_ENABLE=ON \
-        -DCMAKE_INSTALL_PREFIX="${MY_OPENVINO_BASEDIR}/openvino_dist"
+        -DCMAKE_INSTALL_PREFIX="${OPENVINO_BASEDIR}/openvino_dist"
     
     make -j 4
     make install
 
-If you would like to use a specific version of Python, or use a virtual environment you can set the `PYTHON_EXECUTABLE` 
-variable. Examples: 
+The Python module is installed in the `${OPENVINO_BASEDIR}/openvino_dist/python/python<version>/` folder. 
+Set up the OpenVINO™ environment in order to add the module path to `PYTHONPATH`:
+
+    source ${OPENVINO_BASEDIR}/openvino_dist/bin/setupvars.sh
+
+If you would like to use a specific version of Python, or use a virtual environment, you can set the `PYTHON_EXECUTABLE` 
+variable. For example: 
 
 ```
 -DPYTHON_EXECUTABLE=/path/to/venv/bin/python
 -DPYTHON_EXECUTABLE=$(which python3.8)
-```
+```   
 
-### Build nGraph Python wheel
+#### Build an nGraph Python Wheel
 
-When OpenVINO is built and installed, we can build the Python wheel by issuing the following command:
+You can build the Python wheel running the following command:
```

Comment:
Consider changing the line to the following:

To build the OpenVINO™ as a Python wheel, use the following steps:
1. Build the Inference Engine libraries with CMake using the instructions in the [Configure and Build as a part of OpenVINO™ Toolkit on Linux and macOS](#configure-and-build-as-a-part-of-openVINO-toolkit-on-linux-and-macOS) section.
2. Move to the `openvino/ngraph/python` folder in the workspace directory and run the `setup.py` command:

Reply:
Proposed change is not requirement for building Python wheel any more.